synapse/client/nodes/signal_config.py

```diff
@@ -0,0 +1,95 @@
+from types import NoneType
+from typing import List
+from synapse.api.nodes.signal_config_pb2 import ElectrodeConfig, PixelConfig, SignalConfig
+from synapse.client.channel import Channel
+
+class Electrodes:
+    """
+    A collection of electrodes.
+    """
+    def __init__  (
+      self,
+      channels: List[Channel],
+      low_cutoff_hz: float,
+      high_cutoff_hz: float
+    ):
+        self.channels = channels
+        self.low_cutoff_hz = low_cutoff_hz
+        self.high_cutoff_hz = high_cutoff_hz
+    
+    def to_proto(self) -> ElectrodeConfig:
```

Comment:
General note: Would be great if these methods (and others) could return a proper Status obj; there is one in this repo, but it is the DeviceStatus, and does not have all codes that would capture method failure. Will probably add in future PR.